streams.cc

```diff
@@ -192,6 +192,10 @@ make_stream(const json& config, default_seed_source& seeder, const std::size_t o
 #ifdef BUILD_block
     else if (type == "block")
         return std::make_unique<block::block_stream>(config, seeder, osize);
+#endif
+#ifdef BUILD_hash_functions
+    else if (type == "block")
```

Comment:
There shouldn't be block. 

Also in my fix I wrote there two lines one for sha3 and one for others. Though, I think there should be only one line with hash_function which will lead to unified stream implementation with unified factory.